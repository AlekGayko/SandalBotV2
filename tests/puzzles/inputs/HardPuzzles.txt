5000 1b6/1B2pQ2/8/4k1B1/3r4/1N1R4/5n2/1K5b w - - 0 1
5000 8/2K1Qp1b/2p5/1k6/8/RBp3r1/8/1R6 w - - 0 1
5000 8/8/8/2N5/B1p1Q3/4n3/p3R3/B1kn2K1 w - - 0 1
5000 2R5/kpb1P1P1/1p6/1P6/K7/8/8/8 w - - 0 1
5000 8/pk1B4/p7/2K1p3/8/8/4Q3/8 w - - 0 1
5000 1q5k/4R3/8/8/1p6/1B6/5R2/1K6 w - - 0 1
5000 4r1b1/1p4B1/pN2pR2/RB2k3/1P2N2p/n1p3b1/3P1p1r/5K1n w - - 0 1
5000 7k/8/5N1P/8/2p5/2N5/8/3K3R w - - 0 1
5000 8/1Q1K3n/1B6/1k6/8/8/1P6/8 w - - 0 1
5000 8/1p3K1p/8/5p2/2Q2P2/k1P4B/3R4/1q6 w - - 0 1
10000 k1K5/p7/2N5/1P6/4pP2/2p1P3/pp6/r3Q3 w - - 0 1
10000 n1N3br/4Bpkr/1pP2R1b/pP1pnPpR/Pp4P1/1P6/1K1P4/8 w - - 0 1
15000 b1B1k3/pr6/8/P6p/1P6/2P2q2/1K6/8 b - - 0 1
5000 3K2B1/1p6/1r6/rk2N3/b1p5/1pP5/1P3P2/8 b - - 0 1
5000 r1b2nrk/pp3ppp/1q2p3/2bpn1N1/5N2/2PQ4/PPB2PPP/R1B2RK1 w - - 0 1
5000 2kr4/K1pp4/1p6/8/8/8/7Q/3R4 w - - 0 1
5000 N4Q2/nk1P4/8/8/4K3/8/8/8 w - - 0 1
5000 b7/N3B3/Q7/3kpp2/3p4/1R4P1/8/7K w - - 0 1
7000 r1bk3r/pppp4/3N4/3N2qn/2Kbppp1/P7/1PPPB1P1/R1BQ3R w - - 0 1
10000 rnb2rk1/pp3ppp/1b6/3q4/3pN3/Q4N2/PPP2KPP/R1B1R3 w - - 0 1
5000 8/4p3/7B/4p3/4k1P1/8/5K2/3R4 w - - 0 1
5000 kbK5/pp6/1P6/8/8/8/R7/8 w - - 0 1
10000 N1bk3r/P5pp/3b1p2/3B4/R2nP1nq/3P3N/1BP3KP/4Q2R b - - 0 1
10000 r2q1bkr/p4p1p/1p3p1B/2b1n3/2ppN3/2P2N2/PP3PPP/R2QR1K1 w - - 0 1
5000 7k/R7/3R1K2/r4p2/5P2/8/6r1/8 w - - 0 1
10000 1n2qr1k/pb3p1r/1p1p4/2pPpNQB/2P1n3/P1P5/5P2/3RKB1R w - - 0 1
5000 8/p5Q1/2ppq2p/3n1ppk/3B4/2P2P1P/P5P1/6K1 w - - 0 1
10000 Q1b2rk1/2p1bppp/p7/1p2n3/P4n2/1BPq4/1P1P1PPP/RNB2RK1 b - - 0 1
5000 5rk1/pbr2pbp/1q2p1pQ/3p4/1P1N4/P2BP3/1B3PPP/5RK1 w - - 0 1
10000 r1r2knR/5p2/p3bPp1/1p2P3/1q1p4/2NB2R1/PP1Q1P2/1K6 w - - 0 1
5000 r2qk2r/ppp2pp1/8/4N3/3bQ3/2P3p1/PP3PP1/RNB2RK1 w - - 0 1
5000 r7/ppp1kp2/1bnp1n2/P3p3/1PB1P3/2PP3b/5P1P/RN2R1K1 b - - 0 1
5000 2r3k1/Q4Rpp/p1q1n3/P2pP3/8/1pP5/1P4PP/5RK1 w - - 0 1
7000 5r1k/1p4pp/p7/1q1QN3/8/1P6/P3p1PP/6K1 w - - 0 1
15000 8/8/8/6pp/6pk/1R6/6KP/8 w - - 0 1
5000 4r1k1/pp4pp/2p5/3p3n/3P2n1/1PP2R2/P1B3P1/R1B1r1NK b - - 0 1
5000 rnbq1knr/ppb4p/3NpPp1/3p4/1P1p1Q2/P7/2PB1PPP/R2QKBNR w KQ - 0 1
