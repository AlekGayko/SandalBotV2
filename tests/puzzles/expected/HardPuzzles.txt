b7e4 M2
c7d6 M2
e4b7 M2
c8h8 M3
d7h3 M3
f2a2 M3
b5d3 M3
c3b1 M4
b7a6 M4
h3f5 M4
e1c3 M5
c8d6 M4
b7g7 M6
a5a8 Material Advantage
d3h7 M3
h2d6 M2
d7d8 M3
a6a2 M2
d6f7 Knight Fork
e4f6 M4
h6c1 M3
a2a6 M2
h4h3 M5
f3e5 Material Advantage
d6d8 M5
g5g7 M5
g2g4 Material Advantage
f4e2 M5
h6g7 M3
h8g8 M6
e5g6 Material Advantage / Defense
a8g8 M3
f7g7 M4
e5f7 M4
b3h3 M4
h5g3 M3
f4h6 M3
